go.mod

```diff
@@ -1,50 +1,189 @@
 module github.com/benchkram/bob
 
-go 1.16
+go 1.17
```

Comment:
Just curios, it doesn't work with go 1.18?

Reply:
It does, but the minimum required is 1.17.